src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -990,14 +990,14 @@ public bool CreateModel(string org, string service, ServiceMetadata serviceMetad
         }
 
         /// <summary>
-        /// Gets the content of the service model as string
+        /// Gets the content of the ServiceModel as string
```

Comment:
I do not like this class reference in the comments. I would prefer it as it was.